css/selectors/cue.json

```
@@ -17,14 +17,10 @@
             },
             "firefox": {
               "version_added": "55",
-              "notes": [
-                "Firefox currently does not support a parameter on <code>::cue</code>.",
-                "Correct limitations to the CSS properties permitted within <code>::cue</code> were implemented in Firefox 69. See <a href='https://developer.mozilla.org/docs/Web/CSS/::cue#Permitted_properties'>Permitted properties</a> for a list of the allowed properties."
-              ]
+              "notes": "Correct limitations to the CSS properties permitted within <code>::cue</code> were implemented in Firefox 69. See <a href='https://developer.mozilla.org/docs/Web/CSS/::cue#Permitted_properties'>Permitted properties</a> for a list of the allowed properties."
```

Suggesting some revisions here because "within ::cue" is ambiguous given the argument situation.

```suggestion
              "notes": "From Firefox 69, only allowed properties apply to the <code>::cue</code> pseudo-element with no argument. See <a href='https://developer.mozilla.org/docs/Web/CSS/::cue#Permitted_properties'>Permitted properties</a> for a list of the allowed properties."
```